### ISSUE TITLE
Can I make the Minting dapp work with the dynamic price contract?

### ISSUE THREAD

**Issue by alessandrohk** [HashLips/hashlips_nft_minting_dapp#57]
I have used the dynamic price contract, but now i realised that the price on the dapp is not dynamic.
is there any way to make the price on the dapp change following the price brackets that i input in the contract?

Any help is very appreciated

**Comment by tintuhd** [comment]
@alessandrohk did you find how to do this?

**Comment by enazaz** [comment]
@alessandrohk @tintuhd Have you guys found any solutions to this? Would greatly appreciate it!

**Comment by tintuhd** [comment]
@enazaz yes! you remove the variables of the Gas limit that way metamask does the pricing

**Comment by thekesslereffect** [comment]
I tried removing the gasLimit variables but in the local host it still says "site suggested" and gives it a really low gas price compared to normal. It also doesnt update when you wait around for the gas to refresh. 
<img width="237" alt="image" src="https://user-images.githubusercontent.com/24286691/155286451-09d41362-4963-4d12-bd6a-1cf113acc961.png">

**Comment by USBA** [comment]
Anyone found a solution to display the current price from the contract instead of showing the static price in the config.json file?

**Comment by Tobias-GH-Schulz** [comment]
Any updates on this from anyone?

**Comment by USBA** [comment]
Solution is to use the minting dapp in [https://github.com/hashlips-lab/nft-erc721-collection](https://github.com/hashlips-lab/nft-erc721-collection). It displays everything from the actual contract.
